Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2032,6 +2049,12 @@ private void ConstructWithGenesisBlock()
                 .Add(storeFixture.Address2)
                 .Add(storeFixture.Address3);
 
+            var validatorPrivKey = new PrivateKey();
+            var systemActions = new IAction[]
+            {
+                new SetValidator(new Validator(validatorPrivKey.PublicKey, BigInteger.One)),
+            };
```

Comment:
Seems better to use validator private keys on `TestUtils`

Reply:
I think it's better to create a new key every time cause it makes this test more secure. Hard coded value may leads some corner cases, not in this case though.

Reply:
What I was concerned was the number of validators. We don't prevent single validator, but I think it's abnormal. Should `ValidatorSet` have to set spec for number of validators?